rs/nns/governance/canister/tests.rs

```diff
@@ -0,0 +1,86 @@
+use super::*;
+use candid_parser::utils::{service_equal, CandidSource};
+
+/// See analogous comments in registry/canister/canister/tests.rs (yes, canister occurs twice).
```

Comment:
Since the test is already duplicated (which I think is fine), why not duplicate the comments? Seems like an unnecessary hop.

Reply:
Done.

I do not have a strong preference. FWIW, I originally copied the comment. I switched because the comment is kinda long, and I thought people don't like copies. I'm not too worried about the copies getting out of sync. I don't think anyone will ever care to significantly change them, but even if they do, and they don't make changes to all copies, the results would not be too horrible.